virtio-devices/src/mem.rs

```diff
@@ -356,11 +360,11 @@ pub struct Resize {
 }
 
 impl Resize {
-    pub fn new() -> io::Result<Self> {
+    pub fn new(size: u64) -> io::Result<Self> {
```

Comment:
When you call it you pass in `hotplugged_size` what's the meaning of that? Perhaps this `size` should also be called the same to disambiguate from the region size?

Reply:
Yes I can update the `Resize` structure along with the `ResizeSender` structure to reflect that.